Sources/Build/BuildPlan.swift

```diff
@@ -576,13 +576,21 @@ public final class SwiftTargetBuildDescription {
         guard let bundlePath = self.bundlePath else { return }
 
         let stream = BufferedOutputByteStream()
+
+        let mainPath: String = AbsolutePath(Bundle.main.bundlePath)
+                                  .appending(component: bundlePath.basename)
+                                  .pathString
+                                  .replacingOccurrences(of: "\\", with: "\\\\")
```

Comment:
I suggest something like this instead:

    var asSwiftStringLiteralContent: String {
        var result = ""
        for us in self.unicodeScalars {
            result += us.escaped(asASCII: false)
        }
        return result
    }

This will also properly escape double-quotes and other problematic characters.

Reply:
I think I slightly prefer this:

```
extension AbsolutePath {
  fileprivate var asSwiftStringLiteralConstant: String {
    return sself.pathString.unicodeScalars.map { $0.esacped(asASCII: false) }
  }
}
```

if it takes care of the backslash problem.